build/tasks/aria-supported.js

```diff
@@ -32,6 +32,46 @@ module.exports = function(grunt) {
 				return [...out, ...Object.keys(rule.props)];
 			}, []);
 			const aQaria = new Set(axe.utils.uniqueArray(roleAriaKeys, ariaKeys));
+			let footnotes;
+
+			/**
+			 * Parse a list of unsupported exception elements and add a footnote
+			 * detailing which HTML elements are supported.
+			 * @param {Array<String|Object>} elements List of supported elements
+			 */
+			const generateSupportFootnote = elements => {
```

Comment:
Given that this is an extracted function, I do not see a reason for `footnotes` to be a top level variable.

I think we should scope `footnotes` to be within this function & return it, lying closer to the analogy of a pure function.